miner/miner_test.go

```diff
@@ -158,26 +158,21 @@ func createMiner(t *testing.T) (*Miner, *event.TypeMux) {
 	if err != nil {
 		t.Fatalf("can't create new chain config: %v", err)
 	}
-	// Create event Mux
-	mux := new(event.TypeMux)
 	// Create consensus engine
-	engine := ethash.New(ethash.Config{}, []string{}, false)
-	engine.SetThreads(-1)
-	// Create isLocalBlock
-	isLocalBlock := func(block *types.Block) bool {
-		return true
-	}
+	chainConfig.Clique = &params.CliqueConfig{Period: 1, Epoch: 30000}
```

Comment:
You can use the existing `chainConfig` instead of initializing a new value. 

```diff
-       genesis := core.DeveloperGenesisBlock(15, common.HexToAddress("12345"))
+       genesis := core.DeveloperGenesisBlock(1, common.HexToAddress("12345"))
        chainConfig, _, err := core.SetupGenesisBlock(chainDB, genesis)
        if err != nil {
                t.Fatalf("can't create new chain config: %v", err)
@@ -161,8 +160,7 @@ func createMiner(t *testing.T) (*Miner, *event.TypeMux) {
        // Create event Mux
        mux := new(event.TypeMux)
        // Create consensus engine
-       engine := ethash.New(ethash.Config{}, []string{}, false)
-       engine.SetThreads(-1)
+       engine := clique.New(chainConfig.Clique, chainDB)
        // Create isLocalBlock
```

Reply:
Applied your suggestion and squashed it all together